faucet/config_parser.py

```diff
@@ -232,3 +236,75 @@ def _watcher_parser_v2(conf, logname):
             result.append(watcher)
 
     return result
+
+
+def write_yaml_file(yaml_, filename, logger):
+    """Overwrites the specified (in data) configurations.
+    :param top_level the name of the top level dict e.g. 'acls', 'dps', ...
+    :param data list of LocusCommentedMap object to overwrite part of the current config.
+        The top map object specifies the config_file.
+    """
+    with open(filename, 'w') as _file:
+        yaml.dump(yaml_, _file, default_flow_style=False)
+        logger.info('written yaml file')
+        os.fsync(_file.fileno())
+        logger.info('fsync-ed yaml file')
+
+def load_acls(config_path):
```

Comment:
this function name doesn't do what it says. (it returns the whole config). the calling code can easily be changed so it receives only the acls.